sa/proto/sa.proto

```diff
@@ -11,12 +11,10 @@ import "google/protobuf/duration.proto";
 // StorageAuthorityReadOnly exposes only those SA methods which are read-only.
 service StorageAuthorityReadOnly {
   rpc CountCertificatesByNames(CountCertificatesByNamesRequest) returns (CountByNames) {}
-  rpc CountFQDNSets(CountFQDNSetsRequest) returns (Count) {}
   rpc CountInvalidAuthorizations2(CountInvalidAuthorizationsRequest) returns (Count) {}
   rpc CountOrders(CountOrdersRequest) returns (Count) {}
   rpc CountPendingAuthorizations2(RegistrationID) returns (Count) {}
   rpc FQDNSetExists(FQDNSetExistsRequest) returns (Exists) {}
-  rpc FQDNSetTimestampsForWindow(CountFQDNSetsRequest) returns (Timestamps) {}
```

Comment:
Why is this being removed from the SARO service?

Reply:
Oops, this was not supposed to be removed. Thanks for catching.

I was surprised that the removal didn't break any tests. I guess the only place that calls this (the RA) doesn't actually call it via the SARO service, it calls it via the SA service. And the implementation of the SA service, `SQLStorageAuthority`, embeds `SQLStorageAuthorityRO`, which does still have the implementation function.

And, conversely, there's no requirement that `SQLStorageAuthorityRO` implement _only_ methods from the SARO interface, so from that perspective the seemingly-dangling implementation of `FQDNSetTimestampsForWindow` is nbd.